packages/react-ui/components/CurrencyInput/CurrencyInput.tsx

```diff
@@ -185,6 +185,10 @@ export class CurrencyInput extends React.Component<CurrencyInputProps, CurrencyI
     this.setState({ selection: normilized });
   };
 
+  private isCursorMoves = (action: unknown) => {
```

Comment:
```suggestion
  private isCursorMoves = (action: number) => {
```

---

И в [`CurrencyInputKeyboardActions.tsx`](https://github.com/skbkontur/retail-ui/blob/4df276e0443f2cea42ef67047fb184f3fa937790/packages/react-ui/components/CurrencyInput/CurrencyInputKeyboardActions.tsx#L24) прокинуть тип:
```typescript
/* 24 */ export const extractAction = new KeyboardActionExctracterBuilder<number>()
```

Reply:
> Работает довольно непредсказуемо	![ezgif com-gif-maker (11)](https://user-images.githubusercontent.com/2708211/114546460-9a13e080-9c76-11eb-84be-8737badc178e.gif)
> Также не работают клавиши `End` и `Home`.

Блин, думал что end и home заработают если их добавить тоже в проверку как и экшны стрелок, но нет, они так и не работают